web-common/src/features/dashboards/stores/dashboard-store-defaults.ts

```diff
@@ -129,13 +133,21 @@ export function getDefaultMetricsExplorerEntity(
   const metricsExplorer: MetricsExplorerEntity = {
     name,
     visibleMeasureKeys: metricsView.defaultMeasures?.length
-      ? new Set(metricsView.defaultMeasures)
+      ? new Set(
+          metricsView.defaultMeasures
+            .map((dm) => normaliseName(dm, metricsView.measures ?? []))
+            .filter((dm) => !!dm) as string[],
+        )
       : new Set(defaultMeasureNames),
     allMeasuresVisible:
       !metricsView.defaultMeasures?.length ||
       metricsView.defaultMeasures?.length === defaultMeasureNames.length,
     visibleDimensionKeys: metricsView.defaultDimensions?.length
-      ? new Set(metricsView.defaultDimensions)
+      ? new Set(
+          metricsView.defaultDimensions
+            .map((dd) => normaliseName(dd, metricsView.dimensions ?? []))
```

Comment:
Nit: Instead of using `enitities ?? []` everywhere, maybe handle `undefined` entities in `normaliseName` directly

Reply:
Good point. Updated